packages/manager/modules/hub/src/components/user-panel/template.html

```diff
@@ -0,0 +1,45 @@
+<div class="manager-hub-user-panel">
+    <div class="mb-3">
+        <ovh-manager-hub-user-infos
+            data-me="$ctrl.me"
+            data-support-level="$ctrl.supportLevel"
+        >
+        </ovh-manager-hub-user-infos>
+    </div>
+    <div class="mb-4">
+        <ovh-manager-hub-payment-mean> </ovh-manager-hub-payment-mean>
+    </div>
+    <div class="mb-4">
+        <ovh-manager-hub-shortcuts data-notifications="$ctrl.notifications">
+        </ovh-manager-hub-shortcuts>
+    </div>
+    <div class="manager-hub-user-panel_links">
+        <strong data-translate="hub_links_title"></strong>
+        <hr />
+        <a data-ng-href="{{ $ctrl.RedirectionService.getURL('help') }}">
+            <span class="oui-icon oui-icon-lifebuoy_concept"></span>
+            <span data-translate="hub_links_help_center"></span>
+        </a>
+        <div data-ng-if="$ctrl.hasChatbot">
+            <hr />
+            <button
+                class="btn btn-link"
+                type="button"
+                data-ng-click="$ctrl.openChatbot()"
+            >
+                <span class="oui-icon oui-icon-speech-bubble_concept"></span>
+                <span data-translate="hub_links_chatbot"></span>
+            </button>
+        </div>
+        <hr />
+        <a data-ng-href="{{ $ctrl.RedirectionService.getURL('tasks') }}">
+            <span class="oui-icon oui-icon-traffic-cone_concept"></span>
+            <span data-translate="hub_links_tasks"></span>
+        </a>
+        <hr />
+        <a data-ng-href="{{ $ctrl.RedirectionService.getURL('support') }}">
+            <span class="oui-icon oui-icon-user-support_concept"></span>
+            <span data-translate="hub_links_create_ticket"></span>
+        </a>
+    </div>
+</div>
```

Comment:
Some filters might be needed for some links, I'm not sure all of them are available to all subs